tests/30rooms/05aliases.pl

```diff
@@ -226,6 +226,37 @@ sub _test_can_create_and_delete_alias {
       })
    };
 
+test "Users with sufficient power-level can delete other's aliases",
+   requires => [ $creator_fixture, $room_fixture, local_user_fixture(), room_alias_fixture(),
+                 qw( can_create_room_alias )],
+
+   do => sub {
+      my ( $user, $room_id, $other_user, $room_alias ) = @_;
+      my $server_name = $user->http->server_name;
```

Comment:
seems unused?

Reply:
Seems to be defined in most of the tests in this file and unused. I'll clean it up.